requirements/constraints.txt

```diff
@@ -9,11 +9,13 @@
 # linking to it here is good.
 
 # use edx-platform package versions.
+celery>=5.2.2,<=5.2.7
+# required for celery>=5.2.0;<5.3.0
+click>=8.0,<=8.1.3
```

Comment:
@alexjmpb Why isn't this requirement fixed to the 8.1.3 version being used on edx-platform?